core/test/Time_test.cpp

```diff
@@ -202,6 +202,27 @@ TEST_CASE("Durations", "[Duration]")
 
     Duration maybeADay = tt_day - tt;
     REQUIRE(maybeADay.seconds() == 1 * days);
+
+    Duration tenSeconds(10.);
+    REQUIRE(tenSeconds.seconds() == 10);
+
+    Duration hundredSeconds(100.);
```

Comment:
Issue (optional): We probably don't get much extra with some of these new tests --- unless you're aware of some behaviour that occurs at these milestones (please add comments above each one if so). Maybe using [0, 10, 1e4] is sufficient to test the new constructor, even [0, 10]. Could add implicit casting (*if* we *need* to lock that down) otherwise fine

Reply:
I have changed these to test parsing different forms of the string representation of doubles. I think their existence is now justified.